plot/plot-eval-validate-cer.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import argparse
+import matplotlib
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+## from scipy.interpolate import UnivariateSpline
+
+arg_parser = argparse.ArgumentParser(
+    '''Creates plot from Training and Evaluation Character Error Rates''')
+    
+arg_parser.add_argument('-m', '--model', nargs='?',
+                        metavar='MODEL_NAME', help='Model Name', required=True)
+                        
+arg_parser.add_argument('-v', '--validatelist', nargs='?', metavar='VALIDATELIST',
+                        help='Validate List Suffix', required=True)
+                        
+arg_parser.add_argument('-y', '--ymaxcer', nargs='?', metavar='Y_MAX_CER',
+                        help='Maximum CER % for y axis', required=True)
+
+args = arg_parser.parse_args()
+
+maxticks=5
+maxCER=int(args.ymaxcer) #max y axis to display
+
+ytsvfile = "tmp-" + args.model + "-" + args.validatelist + "-iteration.tsv"
+ctsvfile = "tmp-" + args.model + "-" + args.validatelist + "-checkpoint.tsv"
+etsvfile = "tmp-" + args.model + "-" + args.validatelist + "-eval.tsv"
+vtsvfile = "tmp-" + args.model + "-" + args.validatelist + "-validate.tsv"
+plotfile = "../data/" + args.model + "/plot/" + args.model + "-" + args.validatelist + "-cer.png"
```

Comment:
Hard-coding the values in Python redundantly instead of passing the fully defined path names from the makefile is really bad style.